functions/methods.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+
+def read_the_file(file_path: str):
```

Comment:
Please add returned type for all functions

Reply:
solved